packages/plugins/plugin-assistant/src/stories/components/MailboxContainer.tsx

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import React, { useMemo } from 'react';
+
+import { Surface } from '@dxos/app-framework';
+import { Filter } from '@dxos/echo';
+import { Mailbox } from '@dxos/plugin-inbox/types';
+import { useQuery } from '@dxos/react-client/echo';
+
+import { type ComponentProps } from './types';
+
+export const MailboxContainer = ({ space }: ComponentProps) => {
+  const [mailbox] = useQuery(space, Filter.type(Mailbox.Mailbox));
+  const data = useMemo(() => ({ subject: mailbox }), [mailbox]);
+
+  // TODO(wittjosiah): Fix styles to fill the container.
+  return (
```

Comment:
i'll redo the DefaultStory "stack".
put anything here for now so that something shows up. e.g., fixed height.